src/main/java/se/sundsvall/installation/api/model/MetaDataEmbeddable.java

```diff
@@ -0,0 +1,34 @@
+package se.sundsvall.installation.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Getter
+@Setter
+@Builder(setterPrefix = "with")
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+public class MetaDataEmbeddable {
```

Comment:
To me the word `Embeddable` is connected to the DB model, maybe just call the class for InstallationMetaData?